client/app/components/visualizations/editor/TabbedEditor.less

```diff
@@ -0,0 +1,5 @@
+.visualization-editor-tabs {
+  .ant-tabs-tab {
+    padding: 12px 16px;
```

Comment:
Just a question: why was this needed? As far as I remember, we didn't change this anywhere.

Reply:
I have no direct idea (may be that we changed ant-tab style for all Redash), but without it, it looks like this on a blank project:

![image](https://user-images.githubusercontent.com/3356951/80101598-ffdbf980-8547-11ea-9576-6da1a65c4a4a.png)

Reply:
Nevermind, my bad, just noticed I had antd v4 on that blank page :grimacing: